.github/workflows/main.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
drop 3.9?

Reply:
Soon (_next release_)

_Note: Python 3.9 is still supported by starknet.py_